content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -130,21 +130,27 @@ See [Network Requirements](./network-requirements.html) for details.
 
 ### Software Requirements (Standalone Deployment)
 
-Some operating systems have specific configuration requirements:
+The following list provides the general requirements of Terraform
+Enterprise:
+
+- A 64-bit architecture Linux-based operating system
+- Linux Kernel 3.10 or greater
+- Docker Engine 17.06.2-ce to 20.10:
+  - Docker Engine 18.01.0-ce or greater requires Replicated 2.32.0
+    or greater
+  - In Online mode, the installer will install Docker Engine
+    automatically
+  - In Airgapped mode, Docker Engine must be installed before running
+    the installer
+- The faccessat2 syscall:
```

Comment:
is it worth to direct customers the reason for this requirement like in the release notes?
https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2

I had no idea what the faccessat2 syscall is/does.

Reply:
How does https://github.com/hashicorp/terraform-website/pull/1980/commits/4816fba670c6b420c4185f866d7dd08fff6417ba look?